scripts/tokenomics-emulation/index.ts

```diff
@@ -88,6 +89,8 @@ async function main() {
     new EmulationsFactory(network, account, ddcContract, cereContract, batcher),
   );
   await emulations.run();
+  const validator = new ValidatorsEmulation(config, account);
```

Comment:
Include it in the common emulations flow and add a section in `config.json`.

Reply:
added